src/panels/media-browser/hui-dialog-browser-media-player.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { fireEvent } from "../../common/dom/fire_event";
+import { createCloseHeading } from "../../components/ha-dialog";
+import "../../components/ha-hls-player";
+import type { HomeAssistant } from "../../types";
+import { MediaPlayerBrowserDialogParams } from "./show-media-player-dialog";
+
+@customElement("hui-dialog-browser-media-player")
+export class HuiDialogBrowserMediaPlayer extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false })
+  private _params?: MediaPlayerBrowserDialogParams;
+
+  @internalProperty() _sourceUrl?: string;
+
+  @internalProperty() _sourceType?: string;
+
+  @internalProperty() _title?: string;
+
+  public showDialog(params: MediaPlayerBrowserDialogParams): void {
+    this._params = params;
+    this._sourceUrl = this._params.sourceUrl;
+    this._sourceType = this._params.sourceType;
+    this._title = this._params.title;
```

Comment:
Why do we need these 3 separate parameters? We can just use `this._params.sourceUrl`?

Reply:
What do you mean? Just have an object? `this._param.source`

I am using each of those params

Reply:
Yeah, but they are already in `this._param`? And we don't change them so...

Reply:
gotcha
that makes sense